src/databricks/labs/ucx/source_code/linters/context.py

```diff
@@ -12,7 +12,7 @@
     PythonLinter,
     SqlLinter,
 )
-from databricks.labs.ucx.source_code.linters.dbfs import DbfsUsageSqlLinter, DBFSUsagePyLinter
+from databricks.labs.ucx.source_code.linters.dfsa import DfsaPyLinter, DfsaSqlLinter
```

Comment:
please rename `dfsa` in code to `directfs` (or `directfsaccess`, though it's more verbose), as it's more clear for outside devs what does it mean.

Reply:
done